modules/go-release-notes-1.3.1.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+[id="gitops-release-notes-1-3-1_{context}"]
+= Release notes for {gitops-title} 1.3.1
+
+{gitops-title} 1.3.1 is now available on {product-title} 4.7, 4.8, and 4.9.
+
+[id="fixed-issues-1-3-1_{context}"]
+== Fixed issues
+
+* After upgrading to v1.3.0, the Argo CD pods in OCP clusters running behind a proxy were frequently recreated. This was happening since the operator was not returning an ordered slice of environmental variables which leads to the failure of a reconciler. link:https://issues.redhat.com/browse/GITOPS-1489[GITOPS-1489]
```

Comment:
```suggestion
* Before this update, the Operator was not returning an ordered slice of environment variables after upgrading to v1.3.0. As a result, a reconciler failed, which caused the frequent recreation of Argo CD pods in {product-title} clusters running behind a proxy. link:https://issues.redhat.com/browse/GITOPS-1489[GITOPS-1489]

- Refer to Rolfe's [RN template](https://drive.google.com/file/d/1mmljhJFyLoN0ZcCGYtuF2YKWMg0LZhF4/view) from the [training session doc](https://docs.google.com/spreadsheets/d/1oNNCThdiQfWcPI97x8_CUr5_jL6gvq7LypTlWVYm9Aw/edit#gid=0).
- operator/Operator Refer: [operator-capitalization](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#operator-capitalization)
- OCP/{product-title} Refer: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#product-name-and-version
- environmental variables/environment variables

```

Reply:
Hi, @Srivaralakshmi. It's very helpful to provide supporting information like you've done here. Definitely a best practice! It looks like you might have included your supporting information inside the suggestion bloc.
@DebarghoGhosh For release notes templates, I recommend consulting the following section in the Supplementary Style Guide: https://redhat-documentation.github.io/supplementary-style-guide/#release-notes

Reply:
- Refer to Rolfe's [RN template](https://drive.google.com/file/d/1mmljhJFyLoN0ZcCGYtuF2YKWMg0LZhF4/view) from the [training session doc](https://docs.google.com/spreadsheets/d/1oNNCThdiQfWcPI97x8_CUr5_jL6gvq7LypTlWVYm9Aw/edit#gid=0).
- operator/Operator Refer: [operator-capitalization](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#operator-capitalization)
- OCP/{product-title} Refer: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#product-name-and-version
- environmental variables/environment variables

Reply:
> Hi, @Srivaralakshmi. It's very helpful to provide supporting information like you've done here. Definitely a best practice! It looks like you might have included your supporting information inside the suggestion bloc. @DebarghoGhosh For release notes templates, I recommend consulting the following section in the Supplementary Style Guide: https://redhat-documentation.github.io/supplementary-style-guide/#release-notes

Thanks, @rolfedh. Glad you noticed it and liked it. 
I have moved the supporting information outside now. My bad! Thanks for informing me.